### PR TITLE
Add tests for batchnorm1d special case

### PR DESCRIPTION
**Pull request by houseroad**
Test for https://github.com/pytorch/pytorch/pull/6967

### CODE REVIEW

**Comment by houseroad** [comment]
@onnxbot retest this please
